.github/workflows/sanity.yaml

```diff
@@ -0,0 +1,22 @@
+name: sanity
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    paths:
+      - '**'
+
+jobs:
+  sanity:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '~1.16'
+      - name: Run formatting checks
+        run: make vendor && make format && git diff --exit-code
```

Comment:
do we really care about the vendor directory for `rukpak`? I see the `vendor` target calls `tidy`, which we probably do care about -- can we factor it out?

Reply:
Nope just an oversight - we can only specify `tidy` for now.